tardis/io/config_validator.py

```diff
@@ -331,47 +331,64 @@ def to_type(self, value):
 
 class PropertyTypeQuantity(PropertyType):
     def check_type(self, value):
-        try:
-            quantity_split = value.strip().split()
-            quantity_value = quantity_split[0]
-            quantity_unit = ' '.join(quantity_split[1:])
-            if quantity_unit.strip() == 'log_lsun':
-                quantity_unit = 'erg/s'
+        if isinstance(value, units.Quantity):
```

Comment:
I thought you were speaking about introducing more try/except blocks. Using hasattr instead of isinstance actually sounds good, so I'll do that! Thanks for the suggestion!